examples/adjoints/adjoints_ex6/femparameters.h

```diff
@@ -0,0 +1,149 @@
+
+#ifndef __fem_parameters_h__
+#define __fem_parameters_h__
+
+#include "libmesh/libmesh_common.h"
+#include "libmesh/dof_map.h"
+#include "libmesh/enum_norm_type.h"
+#include "libmesh/function_base.h"
+#include "libmesh/getpot.h"
+#include "libmesh/id_types.h"
+#include "libmesh/parallel_object.h"
+#include "libmesh/periodic_boundaries.h"
+#include "libmesh/periodic_boundary.h"
+
+#include <limits>
+#include <map>
+#include <string>
+#include <vector>
+
+
+
+class FEMParameters : public libMesh::ParallelObject
```

Comment:
Yeah: e.g. take a look at adjoints_ex2/Makefile.am

I'm not sure why, but for some reason using symbolic links worked fine for data files and for source files but gave me trouble I couldn't figure out with header files.